ui/furhat-screen/assets/map/route.html

```diff
@@ -0,0 +1,542 @@
+<!DOCTYPE html>
```

Comment:
Is this automatically generated file? If so, we do not need it in the repository.

Reply:
File removed.